tasks/approve.yml

```
@@ -1,8 +1,19 @@
 ---
+- name: Get total number of current nodes
+  # skip_yaml_lint
+  ansible.builtin.command: |
+    kubectl get nodes --no-headers \
+      --kubeconfig {{ openshift_kubeconfig_file }} | wc -l
+  register: csr_autoapprove_current_nodes
+  changed_when: true
+
 - name: Create async csr approvial job
   environment:
     KUBECONFIG: "{{ openshift_kubeconfig_file }}"
-    NODES_COUNT: "{{ csr_autoapprove_node_count }}"
+    # skip_yaml_lint
+    NODES_COUNT: "{{ csr_autoapprove_current_nodes.stdout
+        - ocp_master_count
+        - csr_autoapprove_node_count }}"
```

NODE_COUNT ist die Anzahl der Nodes, die jetzt hinzugefügt werden.
Müsste das dann nicht sein:
ALLE_NODES - BEREITS_VORHANDEN
also
csr_autoapprove_node_count + ocp_master_count - csr_autoapprove_current_nodes.stdout
Wobei csr_autoapprove_node_count = Worker + Infra

```suggestion
    NODES_COUNT: "{{ csr_autoapprove_node_count
        + ocp_master_count
        - csr_autoapprove_current_nodes.stdout }}"
```

---

oder gleich 
ocp_worker_count + ocp_infra_count + ocp_master_count - csr_autoapprove_current_nodes.stdout